### ISSUE TITLE
How to draw a ℘?

### ISSUE THREAD

**Issue by kokic** [QuarticCat/detypify#5]
How to draw a $\wp$ (`\wp`, ℘) ? At present, it seems that all inputs of the trajectory will be recognized as φ and other results. Is this because the dataset does not contain ℘?

**Comment by QuarticCat** [comment]
Right. Supported symbols are listed in [this file](https://github.com/QuarticCat/detypify/blob/main/assets/supported-symbols.txt). I'm wondering if I should hide help messages inside the question icon on the top right.

**Comment by QuarticCat** [comment]
You can submit drawing samples of this symbol to me through the contrib mode on the web page (click the fire icon). I'll update my dataset and re-train the model at this weekend. New symbols will be supported if it has enough samples.

**Comment by kokic** [comment]
℘ is actually a very special character for typst, which means *"currently unable to find a typst mode input"*. Of course, this is entirely a typst implementation issue. This also means that if detypify wants to handle symbols like ℘, it can only return a `"unicode"` instead of `"typst's ASCII"` in the result.

**Comment by QuarticCat** [comment]
Oh... Then currently detypify doesn't support it. The code assumes all symbols are in the `sym` module (i.e. `#sym.xxx`). Sorry for that. But we can leave this issue open util someday I have spare time and find a good way to support other symbols. There once be a similar issue, see #3.

**Comment by QuarticCat** [comment]
BTW, there's a general unicode recognizer: https://shapecatcher.com/

It's algorithm requires a relatively large amount of disk space, so I cannot use it in detypify. And it works bad on solid shapes. But generally it's a good tool and can be used as a complement of detypify. Maybe I should add a link to it on detypify's website.
